lblod/frontend-gelinkt-notuleren-publicatie
# GN-4552: update ember to 4.12.x

## Overview
This PR includes the following changes:
- An update of `ember-source` (and related packages) to 4.12.3
- An update of `ember-data` to 4.12.4
- A fix for `ember-data` fastboot compatibility (see https://github.com/emberjs/data/issues/8475)

##### connected issues and PRs:
[GN-4552](https://binnenland.atlassian.net/browse/GN-4552?atlOrigin=eyJpIjoiMjVjZmEwMjZjN2M4NDM5Mzg2YzUwNmU0M2RlNTI0ZTUiLCJwIjoiaiJ9)

### How to test/reproduce
- Start the application
- Check if you find any breaking issues (especially with regard to ember-data)

### Challenges/uncertainties
This PR only includes minor updates of packages, there should not be any new issues.


### Checks PR readiness
